index.js

```diff
@@ -94,8 +94,20 @@ var initConfig = function(projectConfig) {
  * The singleton public agent. This is the public API of the module.
  */
 var publicAgent = {
+  // FOR_TESTING_ONLY
+  __DISABLE_START_CHECK__: false,
```

Comment:
Not advocating for one way or the other, but for the sake of uniformity you could consider what I've done in the debug agent: https://github.com/GoogleCloudPlatform/cloud-debug-nodejs/blob/master/src/index.js#L44

It would minimize the surface of the public agent, even if the additions are debug-only.